### DISCUSSION TITLE
Preview 1.10.1_06 with new command line options and improved folding symbols

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#319] [Announcements]
Now you can combine command-line options `-N` (non-interactive) and `-S` (stop) with a new option `-R` (run script) followed by the file path of any script you want to execute at the start. Such scripts run without any security restrictions. They do not have any node variable assigned. Selection is supported only if you start Freeplane with the UI. You can start Freeplane with any number of scripts by repeating the `-R` option as many times as needed.

Changes in folding symbols have been initiated by Rexel in  https://github.com/freeplane/freeplane/discussions/317, but they do not completely follow his suggestions.

Enjoy and share your feedback.

**Comment by abc16361** [comment]
Trying to keep folded nodes look similar to other nodes, I set the size 7 to folding symbol. When a folded node is hovered, the folding symbol changes bigger, signaling it can be unfolded. But on the latest alpha version, a dark and bigger default size of folding symbol is used on selected folded or hovered folded nodes, and **this could make a map full of dark dots when multiple folded nodes are selected.** The old way looks better to me.
![foldingSymbol](https://user-images.githubusercontent.com/101362122/163767255-0519eec9-872a-4a4a-b811-221053d61929.png)

**Reply by dpolivaev** [reply]
Fixed in preview 1.10.1_07

**Reply by abc16361** [reply]
> Fixed in preview 1.10.1_07

Already installed fp-1.10.1.08. It looks good.Thank you.
